CovertInfiltration/Src/CovertInfiltration/Classes/UIChainsOverview.uc

```diff
@@ -113,6 +113,7 @@ simulated function OnInit()
 
 	// Tutorial
 	class'UIUtilities_InfiltrationTutorial'.static.ActivityChains();
+	class'UIUtilities_InfiltrationTutorial'.static.ResistanceInformants();
```

Comment:
Remove `ActivityChains` from here and add it as a "follow-up stage" to `ResistanceInformants` (see `GeoscapeEntry2` for example).

This way, if both are triggered at the same time, the animation/sfx are properly triggered

Reply:
Superseded by #506